proteus/Domain.py

```diff
@@ -28,6 +89,31 @@ def writeXdmf(self,ar):
         """
         raise UserWarning("Xdmf output not implemented")
 
+    def getBoundingBox(self):
+        """
+        Get the bounding box of a set of vertices.
+        """
+        self.x = []
+        self.L = []
+        for d in range(self.nd):
+            xMax = max(row[d] for row in self.vertices)
+            xMin = min(row[d] for row in self.vertices)
+            self.x += [xMin]
+            self.L += [xMax-xMin]
+
+    def setGravity(self, g=None):
```

Comment:
Do you mean setGravity or getBoundingBox?
Domains had their own getBoundingBox previously:
https://github.com/erdc-cm/proteus/blob/master/proteus/Domain.py#L267-L283
https://github.com/erdc-cm/proteus/blob/master/proteus/Domain.py#L726-L747